src/Simplify.elm

```diff
@@ -8433,17 +8395,15 @@ mapAlwaysCompositionChecks :
     -> CompositionIntoCheckInfo
     -> Maybe ErrorInfoAndFix
 mapAlwaysCompositionChecks wrapper checkInfo =
-    case ( checkInfo.earlier.fn, checkInfo.earlier.args ) of
-        ( ( [ "Basics" ], "always" ), [] ) ->
+    case ( ( checkInfo.earlier.fn, checkInfo.earlier.args ), checkInfo.later.args ) of
+        ( ( ( [ "Basics" ], "always" ), [] ), [] ) ->
```

Comment:
Could use a tuple with 3 elements.
```suggestion
    case ( checkInfo.earlier.fn, checkInfo.earlier.args, checkInfo.later.args ) of
        ( ( [ "Basics" ], "always" ), [], [] ) ->
```

Reply:
For performance? I think I prefer the grouping of tuples :shrug:

Reply:
I was thinking mostly for readability :thinking: (though yes, it's one less tuple allocation)
Not sure why you find nested tuples to read better, but sure, I don't mind :+1: